tsdfileapi/db.py

```diff
@@ -233,7 +233,11 @@ def table_update(self, table_name, uri_query, data):
     def table_delete(self, table_name, uri_query):
         sql = self.generator_class(f'"{table_name}"', uri_query)
         with sqlite_session(self.engine) as session:
-            session.execute(sql.delete_query)
+            try:
+                session.execute(sql.delete_query)
+            except sqlite3.OperationalError as e:
```

Comment:
how about adding this exception handling to the `sqlite_session` context manager?

Reply:
yes, It might go in the same function.  `sqlite_session` is to get a session and does not include session.execute. 
It was to understand one of the failed test which gets:
```
[E 201124 08:32:11 db:239] Syntax error?: delete from "user_data" where   json_extract(data, '$.key1') = 7 
[E 201124 08:32:11 api:2392] no such function: json_extract
```

Reply:
aha, go it.